src/components/App.jsx

```diff
@@ -47,22 +64,24 @@ const App = () => {
   }
 
   // act as componentDidUpdate
-  useEffect(() => { getOneProduct(productId) }, [productId])
+  useEffect(() => { getOneProduct(productId) }, [])
```

Comment:
Is there a reason that productId is deleted? I think we need productId in the array to make everything re-render